examples/resourceServer/client/index.html

```diff
@@ -24,7 +24,7 @@
     {{#unless isUrlParamsValid }}
         <p>The URL params are not set in order to authorize an oauth2 token.</p>
         <p>The following link will simulate arriving to this page from another site that wishes to authenticate with oauth2.</p>
-        <a href="/?client_id=resourceServerClient&redirect_uri=http://localhost:3200/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
+        <a href="/?client_id=clientApplication&redirect_uri=http://localhost:3100/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
```

Comment:
The client_id is a good catch and was a mistake. There is a pending pull request that fixes this. Refer to the previous comment about the port number.